src/app/components/Document/index.jsx

```diff
@@ -29,8 +29,14 @@ const Document = ({ assets, data, styleTags, helmet }) => {
         {styleTags}
       </head>
       <body>
-        <AfterRoot />
-        <AfterData data={data} />
+        {/* eslint-disable react/no-danger */
+        /* disabling the rule that bans the use of dangerouslySetInnerHTML until a more appropriate implementation can be implemented */}
```

Comment:
A 'more appropriate implementation' might be refactoring this document to be a string over a React component. It doesn't need to be a React component, and `renderToStaticMarkup` is a slower API than just returning the string. This wasn't considered in this PR as I did not want to change underlying implementations where possible.

Reply:
React v16 released [`renderToNodeStream`](https://reactjs.org/docs/react-dom-server.html#rendertonodestream), which enables a React app to be streamed as a [node stream](https://medium.freecodecamp.org/node-js-streams-everything-you-need-to-know-c9141306be93). Unsure on how this would fit into Mozart, but it would be a possibility from our side if the `Document` was just a string.

Reply:
Stream wouldn't be possible with Mozart but this application ought to be able to handle it (not in this PR).